java/client/test/org/openqa/selenium/edge/EdgeOptionsFunctionalTest.java

```diff
@@ -0,0 +1,123 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.edge;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.openqa.selenium.remote.CapabilityType.ACCEPT_INSECURE_CERTS;
+import static org.openqa.selenium.support.ui.ExpectedConditions.titleIs;
+import static org.openqa.selenium.testing.drivers.Browser.EDGE;
+
+import org.junit.After;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.build.InProject;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.testing.Ignore;
+import org.openqa.selenium.testing.JUnit4TestBase;
+import org.openqa.selenium.testing.NeedsLocalEnvironment;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Base64;
+
+public class EdgeOptionsFunctionalTest extends JUnit4TestBase {
+
+  private static final String EXT_PATH = "third_party/chrome_ext/backspace.crx";
+
+  private EdgeDriver driver = null;
+
+  @After
+  public void tearDown() {
+    if (driver != null) {
+      driver.quit();
+    }
+  }
+
+  @NeedsLocalEnvironment
```

Comment:
Why does it need a local environment?

Reply:
I don't know why Chrome needs it in ChromeOptionsFunctionalTest, but I'll remove it since all tests are passed without local environment.